### PR TITLE
Add records_default_proxied variable

### PR DESCRIPTION
**Pull request by glensc**
## references

Fixes https://github.com/cloudposse/terraform-cloudflare-zone/issues/47

NOTE: df3471677c3846aa5b76084eb8c49d89aa3e341f fixes version pins added in 9141c9a3c840896b99db111f32efec7810dd7df9 commit.

### CODE REVIEW

**Comment by glensc** [comment]
no point resolving conflicts if this isn't going to be accepted. so far nobody as reviewed this

**Comment by glensc** [comment]
well. nobody reviewed the code, what's the point of putting more work if nobody accepts this?

**Comment by glensc** [comment]
@osterman

**Comment by glensc** [comment]
@goruha

**Comment by nitrocode** [comment]
One of the best ways to get visibility is to join the slack channel called #pr-reviews in the sweetops community slack

File: main.tf [hunk 1]
```diff
@@ -10,6 +10,7 @@ locals {
     for index, record in var.records :
     try(record.key, format("%s-%s-%s", record.name, record.type, record.value)) => record
   } : {}
+  records_default_proxied = var.records_default_proxied
```
Comment (nitrocode) [inline main.tf:13]: This local isn't providing much value. You can use the var directly below instead

File: main.tf [hunk 2]
```diff
@@ -40,7 +41,7 @@ resource "cloudflare_record" "default" {
   value    = each.value.value
   comment  = lookup(each.value, "comment", null)
   priority = lookup(each.value, "priority", null)
-  proxied  = lookup(each.value, "proxied", false)
+  proxied  = lookup(each.value, "proxied", local.records_default_proxied)
```
Comment (nitrocode) [inline main.tf:44]:
```suggestion
  proxied  = lookup(each.value, "proxied", var.records_default_proxied)
```

**Comment by glensc** [comment]
I can't reopen the pr

**Pull request closed without merging**